src/components/PeopleManagement/AddMembersModal/AddMembersModalContent.jsx

```diff
@@ -52,7 +52,7 @@ const AddMembersModalContent = ({
         <Col>
           <FormattedMessage
             id="people.management.add.members.modal"
-            defaultMessage="Only members registered with your organization can be added to a group."
+            defaultMessage="Only members registered with your organization can be added to a group. "
             description="Subtitle for the add members modal"
           />
           <Hyperlink
```

Comment:
I believe this `Hyperlink` should be using `variant="inline"` given its an inline hyperlink. Inline hyperlinks per the design system should have the underline text decoration ([Figma](https://www.figma.com/design/I7OtSoBzgms4GxDXfzw0pW/Paragon-Elm-Theme?node-id=42684-81671&t=M1KigQmbyCfja80s-4)).